tests/library/test_json.py

```diff
@@ -638,6 +639,7 @@ def test_bad_with_prefix(
     ):
         schema_obj = {
             "prefixItems": self.prefix_schema_obj,
+            "items": False,
```

Comment:
Why is this needed? If you're only adding capabilities, surely the existing tests should remain the same?

Reply:
Essentially the old test and behavior were *wrong*. According to the json schema specification, ["Omitting this keyword has the same assertion behavior as an empty schema."](https://json-schema.org/draft/2020-12/json-schema-core#section-10.3.1.2-4). Previously, we were treating an omitted "items" keyword as "no items not explicitly provided by prefixItems are allowed", but the behavior should instead be "put anything you want here".

I'll take a look to see if we can be a little more explicit about this behavior in the tests.

Reply:
Understood. So we need to make sure this is added wherever it's needed, and then in the new 'unconstrained' tests that it works both ways - when `"items": True` explicitly, and when `items` is omitted.

Reply:
Added some tests for these too :)